modules/boot/include/settings.h

```diff
@@ -42,7 +42,11 @@ enum GZSettingID : uint32_t {
     STNG_CHEATS_UNRESTRICTED_ITEMS,
     STNG_CHEATS_TRANSFORM_ANYWHERE,
     STNG_CHEATS_DISABLE_ITEM_TIMER,
+#ifdef WII_PLATFORM
     STNG_CHEATS_GALE_LJA,  // Wii only, but we reserve the id anyway
+#endif
```

Comment:
We need to keep the ID the same to make sure the id's are the same for every version, and prefereably also between gc and Wii. If you want to add new setting id's, add them at the end